### DISCUSSION TITLE
PR自动检查格式

### DISCUSSION THREAD

**Discussion by marcosteam** [DIYgod/RSSHub#7671] [Q&A] [answered]
#7670 #7669 

实在不好意思，开了太多次 PR，但是不知道为什么自己的 PR 就是一直都过不了检测，每一次 PR 被 Close 了也不能 Reopen，希望有人帮忙解决一下这个问题。

**Comment by NeverBehave** [comment, marked as answer]
注释中说明：
如果与路由无关, 请写`NOROUTE`
请不要删除代码块`routes`标识 

例如：

![图片](https://user-images.githubusercontent.com/17120571/121224601-98f9cc80-c83d-11eb-819f-84125f1dd301.png)

在你的PR中：

![截屏2021-06-08 上午9 39 19](https://user-images.githubusercontent.com/17120571/121224473-72d42c80-c83d-11eb-8aaf-714eb539ca5d.png)

如果只是格式相关问题，在修正以后通知一下，不需要开新的PR。我们会重新开启并重新检测
